contrib/IECoreUSD/src/IECoreUSD/USDScene.cpp

```diff
@@ -935,20 +247,18 @@ IECoreScene::PointsPrimitivePtr convertPrimitive( pxr::UsdGeomPointInstancer poi
 	}
 
 	pxr::VtVec3fArray pointsData;
-	positions.Get( &pointsData, time );
-
-	IECore::V3fVectorDataPtr positionData;
-	convert( positionData, pointsData );
+	pointInstancer.GetPositionsAttr().Get( &pointsData, time );
+	IECore::V3fVectorDataPtr positionData = DataAlgo::fromUSD( pointsData );
 
 	IECoreScene::PointsPrimitivePtr newPoints = new IECoreScene::PointsPrimitive( positionData );
 
-	IECore::DataPtr orientationData = convertValue( orientations, time );
+	IECore::DataPtr orientationData = DataAlgo::fromUSD( pointInstancer.GetOrientationsAttr(), time );
 	newPoints->variables["orient"] = IECoreScene::PrimitiveVariable( IECoreScene::PrimitiveVariable::Vertex, orientationData );
```

Comment:
As above, I think so, yes, but in another PR. This does raise the question of how much we're willing to change behaviour during this refactor though. I think the st<->uv change is totally defensible as a bug fix - you can't render UVs without it. But here the naming is a bit more arbitrary and folks might be relying on it. This is all in `contrib` of course so we make no guarantees, and I'm inclined to make all the improvements we can, but there are a few brave souls out there using it in production already. Anyone want to make a counter argument?

Reply:
Instead of the counter argument, I'll try to bolster the same side:
- As you say, it was previously impossible to render UVs, so anyone relying on IECoreUSD probably didn't get too far along.
- The goal of your current work is to make loading USD in Gaffer as simple as possible, so outside of predominant/documented Cortex conventions, we should trend towards primvar names that either match USD conventions or work well in Gaffer out of the box.
- Once #1058 is merged, we could tie the primvar renaming to a compatibility version if we felt it was merited to be cautious.

Reply:
> As you say, it was previously difficult to render UVs, so anyone relying on IECoreUSD probably didn't get too far along.

I don't have all the details, but I don't believe this to be true. I think some folks outside the Cs group are using SceneReader/USD in anger (and possibly frustration!).

> Once #1058 is merged, we could tie the primvar renaming to a compatibility version if we felt it was merited to be cautious.

If it was merged then I think that would be the right thing to do. There are a lot of things we can do better once that is merged. In the meantime I think I'll be satisfied to just lock Gaffer 58 on the current Cortex version, and include these changes in 59.